### ISSUE TITLE
CardDAV server sync

### ISSUE THREAD

**Issue by peperunas** [tutao/tutanota#458]
Hello!

I have been trying your service but I could not find any way to import/sync contacts from an external CardDAV server.

Is this going to be implemented?

Thanks!

**Comment by charlag** [comment]
Hi!
It would be a nice feature but there are certain difficulties, e.g. we cannot contact another server from the browser because of CORS, so it is only possible in the app. So I cannot say that it is on a roadmap now but we'll definitely give it a thought later.

**Comment by letkan** [comment]
+1 for this feature.
Meanwhile maybe importing a contacts.vcf would be easier to implement?

**Comment by charlag** [comment]
@letkan isn't vCard import already implemented in the new client?

**Comment by letkan** [comment]
Yes, sorry, that option was so well hidden that I discovered it afterwards :).

**Comment by szaimen** [comment]
+1

**Comment by luk6173** [comment]
I definitely need this option, otherwise I can't see my contacts in various messengers.
